vaas/vaas/cluster/api.py

```diff
@@ -148,3 +153,92 @@ class Meta:
             'name': ['exact'],
             'template': ALL_WITH_RELATIONS
         }
+
+
+class CommandResource:
+    def __init__(self, pk=None, varnish_ids=None, status=None, output=None):
+        self.pk = pk
+        self.id = pk
+        self.varnish_ids = varnish_ids
+        self.status = status
+        self.output = output
+
+    def __repr__(self):
```

Comment:
Did we stopped use type hinting ?

Reply:
I didn't notice that we ever got started in this project.

Reply:
I think we already started. 
**https://github.com/allegro/vaas/pull/410/files#diff-e46cb127a77da803977937c8ed0395e87e5c48ebaf25dd0b80047615142de952R64**

Reply:
I've added typing where is reasonable.